tendrl/monitoring_integration/alert/handlers/cluster/brick_utilization_handler.py

```diff
@@ -0,0 +1,100 @@
+from etcd import EtcdKeyNotFound
+from subprocess import CalledProcessError
+from tendrl.commons.event import Event
+from tendrl.commons.message import ExceptionMessage
+from tendrl.monitoring_integration.alert import constants
+from tendrl.monitoring_integration.alert.handlers import AlertHandler
+from tendrl.monitoring_integration.alert import utils
+from tendrl.monitoring_integration.alert.exceptions import NodeNotFound
+
+
+class BrickHandler(AlertHandler):
```

Comment:
Better in each handler, in `parse_alert_metrics` add a sample json for the alert metrics for reference. Its really tough to visualize what parsing and why its happening?